docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -112,7 +110,52 @@ To send a command from the client and to process it on the server side:
     }
     ```
 
-<a name="servermetadata"></a>
+## Updating and Retreiving Playlist
+
+To create a playlist from the server side:
+
+1. Create a new instance of the [Tizen.Multimedia.Remoting.MediaControlPlaylist](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlPlaylist.html) class:
+
+    ```
```

Comment:
Hello Ms. @mijin85cho,

I have doubt that, based on Markdown recommendations from https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md. Should we add label for C-Sharp language while starting the code block? As these code blocks are for DotNet code and use C-Sharp language only.

Kindly let me know your thoughts regarding the same.

Reply:
Hello, @rishu-samsung 
Thanks for pointing it out. Yes, it is strongly recommended to apply language identifier label for the language type.

Reply:
Done